terraform/dev/eks-cluster.tf

```diff
@@ -0,0 +1,38 @@
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.0.4"
+
+  cluster_name    = local.cluster_name
+  cluster_version = "1.24"
+
+  vpc_id                         = var.vpc_id
+  subnet_ids                     = ["subnet-a3f202fa", "subnet-fc843999", "subnet-0fc91a78"]
+  cluster_endpoint_public_access = true
+  cluster_security_group_id      = var.security_group_id
+  aws_auth_users = [
+    {
+      userarn  = "arn:aws:iam::364159549467:user/liam"
+      username = "liam"
+      groups   = ["system:masters"]
+    }
```

Comment:
It would be better to use a build-specific user than yours, I think.

Reply:
This isn't actually using my user to do the build, it's telling Terraform that my user should have access to the cluster

Reply:
Oooooh interesting, I see! Thanks for the explanation. Is there a way to allow a group of users to have access, then? It seems like it would be good for the whole AWS "Developers" group to have access.

Reply:
Unfortunately, it seems there is not a way to do that.

Reply:
So what does it mean for your user to have access to the cluster? Can you add me and Natalie individually if you can't add the group? I don't want to be in a scenario where you're the only one who can access it, but "access" is a broad term so I'm not sure how big a deal it is.

Reply:
Access in this case means "extract information about the cluster from the EKS web interface or `kubectl`". Theoretically it should be pretty rare that we have to do this, but you and Natalie should definitely have access; I'll add you.

Reply:
You should have access now. This should be verifiable by visiting https://us-west-2.console.aws.amazon.com/eks/home?region=us-west-2#/clusters/dev-eks-ZClsj6PM and confirming that there's no message telling you you lack permissions.

Reply:
Awesome, thank you! Confirmed.